dist.sh

```diff
@@ -30,7 +30,7 @@ echo "... running tests"
 
 for os in linux darwin freebsd windows; do
     echo "... building v$version for $os/$arch"
-    BUILD=$(mktemp -d -t nsq)
+    BUILD=$(mktemp -d -t nsq-XXXXX)
```

Comment:
Does this still work on OSX?

Reply:
We've hit this before, not sure what happened 😝 
thread starts at https://github.com/nsqio/nsq/pull/718#issuecomment-187758182

Reply:
Cool, let's update this to reflect @ploxiln's suggestion in https://github.com/nsqio/nsq/pull/718#issuecomment-187877348 ?